examples/sample-app/application-template-pullspecbuild.json

```diff
@@ -134,7 +134,7 @@
           "sourceStrategy": {
             "from": {
               "kind": "DockerImage",
-              "name": "centos/ruby-27-centos7:latest"
+              "name": "ubi8/ruby-27:latest"
```

Comment:
Is this pulling from docker.io? We should use quay images or `registry.redhat.io` images to avoid throttling flakes.

Reply:
No, that must be automatically resolving to registry.access.redhat.com or registry.redhat.io.  Should I make that explicit?

Reply:
I'd prefer these to be explicit. Builds in particular do not share the registry search list logic that can be set on nodes. At one point we hard-coded `docker.io` as the only registry in the search list (I don't recall if this is still the case).

Reply:
I'm going to let the current test run complete first before repushing with that, just so that we know where we're holding.

Reply:
All the ubi8/ruby-2.7 references should be prefixed with a registry now.